src/backend/parser/cypher_analyze.c

```diff
@@ -72,9 +72,10 @@ static Query *analyze_cypher(List *stmt, ParseState *parent_pstate,
 static Query *analyze_cypher_and_coerce(List *stmt, RangeTblFunction *rtfunc,
                                         ParseState *parent_pstate,
                                         const char *query_str, int query_loc,
-                                        char *graph_name, uint32 graph_oid,
+                                        char *graph_name, Oid graph_oid,
```

Comment:
is this change valid? didnt we shift to uint32 in PG12?

Reply:
It was added back in for clarity, _in some cases_, as there was some confusion with what the actual variable was meant to be used for. Below are some examples.

```
int64 graph_id;
graphid graph_id;
Oid graph_oid;
uint32 graph_oid;
graphid object_graph_oid;
graphid object_graph_id;
```

This was made more complicated by some variables initially being renamed incorrectly in **PG12**, likely by a global search & replace like ` %s/graph_id/graph_oid/g` This caused the two types to get mixed up when `graph_id ` was a suffix to a variable name.